### PR TITLE
2024, March 26th Meeting minute preparation

### PR DESCRIPTION
**Pull request by bigludo7**
#### What type of PR is this?

Add one of the following kinds:

* subproject management

#### What this PR does / why we need it:

Meeting minutes preparation for March 26th meeting.

### CODE REVIEW

**Review by bigludo7 — COMMENTED** [review]
@jlurien For your consideration.
Before next meeting we have to move to LF Wiki for the minutes + send a new invitation series.

**Review by bigludo7 — COMMENTED** [review]
Updating with @fernandopradocabrillo  notes :)

File: documentation/MeetingMinutes/CAMARA Device Location - 2024-03-26.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* [Geofencing - Adding a value in Termination Reason value enum](https://github.com/camaraproject/DeviceLocation/issues/141)
+  - Discussion in progress in Commnonalities: https://github.com/camaraproject/Commonalities/issues/153 
+  - Related issues in [DeviceStatus](https://github.com/camaraproject/DeviceStatus/issues/117) and QoD
+  - Waiting for Commonalities outcomes
+
+
+* [Geofencing API - Add Subscription type 'area-left-or-entered' to subscribe to both event in one time](https://github.com/camaraproject/DeviceLocation/issues/138)
+  - New issue in Commonalities [Subscription-Issue2: Allow event consumers to subscribe to more than one event types with a single subscription](https://github.com/camaraproject/Commonalities/issues/154)
+  - Waiting for Commonalities outcomes
+  - As we have several issues aroung Subscriptions and the progress on commonalities are slow we need to speed up the resolution. One proposal should be to have a cross meeting with Device Status WG as these 2 projets rely on subscriptions.
```
Comment (jlurien) [inline documentation/MeetingMinutes/CAMARA Device Location - 2024-03-26.md:78]:
```suggestion
  - As we have several issues aroung Subscriptions and the progress on commonalities are slow we need to speed up the resolution. One proposal should be to have a cross meeting with Device Status WG as these 2 projects rely on subscriptions.

```

File: documentation/MeetingMinutes/CAMARA Device Location - 2024-03-26.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* [Issue #85](https://github.com/camaraproject/DeviceLocation/issues/85)
+  - As requested by Joachim, issue will remain open until TEF uploads the document with more detail or the information is added to an API_documentation file to keep in the repo for future references.
+    - Joachim: Should be also aligned/synchronysed with Geofencing
+
+## AOB
+
+<p>
+
+- Next call would be on **April 9th, 2024**
+- **Not present in calendar** : Ludovic will check with José about resending a serie - we have probaly to move to LF zoom.
```
Comment (jlurien) [inline documentation/MeetingMinutes/CAMARA Device Location - 2024-03-26.md:124]: Yes, we'll have to move to Confluence and Zoom. I'll prepare next agenda in the Wiki and resend the email with the new link

**Review by jlurien — DISMISSED** [review]

**Review by bigludo7 — COMMENTED** [review]
Approved José corrections

**Comment by jlurien** [comment]
These are already in https://wiki.camaraproject.org/display/CAM/2024-03-26+Device+Location+-+Meeting+Minutes, should we close this PR without merging @bigludo7 ?

**Comment by bigludo7** [comment]
Yes !

**Pull request closed without merging**